Automattic/wp-calypso
# Only get Jetpack Sites for Jetpack Cloud

#### Changes proposed in this Pull Request

* Change `me/sites` endpoint to only request Jetpack sites for Jetpack Cloud

#### Testing instructions

1. Navigate to `/backups`
2. Verify the list of sites is only jetpack and atomic sites
3. Boot Calypso
4. Verify the sites visible are the same as on WordPress.com

#### Next Steps

* Going Directly to a WPCOM site via the site slug ( `/backups/:wpcomSiteSlug`  ) will load up that site. Should we filter the `/sites` endpoint as well?
* The site selector site count is pulled from user info. We need to either provide fields on user info or replace the count.
<img width="663" alt="Screen Shot 2020-04-01 at 10 42 13 AM" src="https://user-images.githubusercontent.com/2810519/78168936-8a2bb480-7405-11ea-9014-dea2c9083a40.png">
